server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -957,6 +958,16 @@ func (e *EventChannel) handleEvent(event *pepb.PublishBuildToolEventStreamReques
 	}
 
 	if e.isFirstStartedEvent(&bazelBuildEvent) {
+		started, _ := bazelBuildEvent.Payload.(*build_event_stream.BuildEvent_Started)
+
+		version, err := semver.NewVersion(started.Started.GetBuildToolVersion())
+		majorVersion := "unknown"
+		if err == nil {
+			majorVersion = strconv.Itoa(int(version.Major()))
+		}
+		metrics.InvocationsByBazelMajorVersionCount.With(
```

Comment:
Should we include the minor version as well? Bazel changes quite a bit, even between minor versions.

Reply:
sgtm, updated